draft-ietf-privacypass-auth-scheme.md

```diff
@@ -172,13 +172,13 @@ origin names, they are delimited with commas "," without any whitespace.
 When used in an authentication challenge, the "PrivateToken" scheme uses the
 following attributes:
 
-- "challenge", which contains a base64url-encoded {{!RFC4648}} TokenChallenge
-value. Since the length of the challenge is not fixed, the base64url data MUST
-include padding. This MUST be unique for every 401 HTTP response to prevent
+- "challenge", which is a quoted string containing a base64url-encoded {{!RFC4648}}
+TokenChallenge value. Since the length of the challenge is not fixed, the base64url
+data MUST include padding. This MUST be unique for every 401 HTTP response to prevent
 replay attacks. This attribute is required for all challenges.
```

Comment:
```suggestion
- "challenge", which contains a base64url-encoded {{!RFC4648}} TokenChallenge
 value. Since the length of the challenge is not fixed, the base64url value MUST
 include padding. As an Authentication Parameter (`auth-param` from {{!RFC9110}}),
 the value can be either a token or a quoted-string, depending on if the base64url string
 includes "=" characters. This challenge value MUST be unique for every 401 HTTP
 response to prevent replay attacks. This attribute is required for all challenges.
```

Reply:
"depending on if the base64url string includes "=" characters" -> "and might be required to ba quoted-string if...."

Reply:
Updated!